src/compose/ComposeBox.js

```diff
@@ -155,8 +164,13 @@ class ComposeBox extends PureComponent<Props, State> {
     isFocused: false,
     isMenuExpanded: false,
     height: 20,
-    topic: this.props.lastMessageTopic,
-    message: this.props.draft,
+    topic:
+      this.props.initialTopic !== undefined
+        ? this.props.initialTopic
+        : isTopicNarrow(this.props.narrow)
+        ? topicOfNarrow(this.props.narrow)
+        : '',
+    message: this.props.initialMessage || '',
```

Comment:
nit: `??` seems a bit more precisely what we mean here, though the behavior is the same; also makes it parallel the `topic` case once that uses `??`